br/pkg/lightning/backend/local/region_job.go

```diff
@@ -592,15 +592,23 @@ func (local *Backend) doIngest(ctx context.Context, j *regionJob) (*sst.IngestRe
 		}
 	}
 
+	var limiter *ingestLimiter
+	if x := local.ingestLimiter.Load(); x != nil {
```

Comment:
we should limit by store, each store will check flow control independently, not a global one